core/domain/exp_domain.py

```diff
@@ -2030,7 +2052,7 @@ def update_states_from_model(
     # incompatible changes are made to the exploration schema in the YAML
     # definitions, this version number must be changed and a migration process
     # put in place.
-    CURRENT_EXP_SCHEMA_VERSION = 54
+    CURRENT_EXP_SCHEMA_VERSION = 55
```

Comment:
Why is this defined both here and in feconf.py

Reply:
removed from feconf.